modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -23,16 +23,16 @@ ContentDashboardAdminManagementToolbarDisplayContext contentDashboardAdminManage
 %>
 
 <clay:row
-	cssClass="no-gutters"
+	cssClass="no-gutters sidebar-wrapper"
 >
 	<clay:container-fluid>
 		<div class="main-content-body">
-			<div class="dashboard-content sheet">
+			<div class="sheet">
```

Comment:
`<clay:sheet>` ? https://github.com/liferay/liferay-portal/blob/master/modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/edit_calendar_booking.jsp#L278

Reply:
`<clay:sheet>` adds `lg` size by default (both in SheetTag and in the ClayComponent), and right now the Tag doesn't accept any size attribute.

In this case I need `sheet` to be as it is, with no size specified (size = false or null) but we can't overload `setSize` method to accept both `String` and `Bool`, so here we'd have three options:

- Don't use the Tag
- Add size to `<clay:sheet>` in the `.tld` and just pass an empty string `<clay:sheet size="">`
- Add size to `<clay:sheet>` in the `.tld` and remove `lg` value by default, which implies changing all actual `<clay:sheet>`uses in portal

Don't know why `lg` is set by default, there should be a good reason for it...

Since this change is not trivial I'll keep the markup, but I think is an interesting think to analyze.

cc @jbalsas

Reply:
> Add size to <clay:sheet> in the .tld and remove lg value by default, which implies changing all actual <clay:sheet>uses in portal

This would cause visual bugs in all the changed `sheet`s

>Don't know why lg is set by default, there should be a good reason for it...

I think it's because it's the most used case in portal

Reply:
Not sure if being the most used case in portal is a good reason since you won't be able to use it with no size at least you pass an empty string as value (which I'm not 100% sure that would work and, if it does, seems a little hacky to me)

Reply:
> Not sure if being the most used case in portal is a good reason [...]

It is, that's the reason behind every **default**.

> [...] since you won't be able to use it with no size [...]

That's a totally different problem that can be addressed with a proper fix, maybe?

Reply:
What kind of proper fix are you thinking about?

Anyway, is this something we want to address on this pr?

Reply:
> What kind of proper fix are you thinking about?

Something like:

> [...] at least you pass an empty string as value (which I'm not 100% sure that would work and, if it does, seems a little hacky to me)

That is not hacky :)

> Anyway, is this something we want to address on this pr?

I wouldn't say that's necessary, no.

Reply:
But if the hacky thing works... maybe that's less tech debt than the raw markup. Just a tradeoff, as usual :)